tests/iterables_test.py

```diff
@@ -63,7 +63,7 @@ def test_sorted_alphanumeric_list():
     ]
 
     sorted_list = sorted_alphanumeric_list(random.sample(test, len(test)))
-    assert all(elem == tester for elem, tester in zip(sorted_list, test))
+    assert all(elem == tester for elem, tester in zip(sorted_list, test, strict=False))
```

Comment:
**suggestion:** use strict zip
```suggestion
    assert all(elem == tester for elem, tester in zip(sorted_list, test, strict=True))
```
I think strict is desired here
It is a test, so a safe change, and we will know right away...